CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Enhancements
+
+- Improve app start integration ([#2266](https://github.com/getsentry/sentry-dart/pull/2266))
```

Comment:
let's mention that this fixes blocking issues such as https://github.com/getsentry/sentry-dart/issues/2233